### PR TITLE
Updates to Managing Airbyte Cloud doc

### PR DESCRIPTION
**Pull request by sophia-wiley**
Updated the [Managing Airbyte Cloud doc](https://docs.airbyte.com/cloud/managing-airbyte-cloud/):
* Added Remove users section
* Edited Sync summary section
* Added Display connection state section

Related to [15795](https://github.com/airbytehq/airbyte/pull/15795) and [16936](https://app.zenhub.com/workspaces/documentation-6266ed73c9cbb400119b1e54/issues/airbytehq/airbyte/16935).

### CODE REVIEW

**Comment by sophia-wiley** [comment]
@marcosmarxm @timroes Please review the info in this [commit](https://github.com/airbytehq/airbyte/pull/16941/commits/217c7af407ab8aeda5abbb2e07e4f272b34817d4).

**Comment by timroes** [comment]
@sophia-wiley I'm not sure if the docs are technically true like this. 0 Bytes will just mean 0 Bytes have been transfered. This could be because it was cancelled, but could also just happen if no bytes have been transferred. Can we just remove this part from the documentation? I feel "0 bytes" should be rather self-explanatory to the user to understand?

**Review by timroes — APPROVED** [review]
LGTM

File: docs/cloud/managing-airbyte-cloud.md [hunk 1]
```diff
@@ -97,76 +106,45 @@ Understanding the following limitations will help you better manage Airbyte Clou
...
 1. On the [Airbyte Cloud](http://cloud.airbyte.io/) dashboard, click **Connections** in the navigation bar.   
 
     The Connections page displays.
 
 2. Click a connection in the list to view its sync history.
 
-3. Under Sync History, a list of syncs for that connection is displayed.
-
-    The sync status is displayed (Succeeded, Cancelled, Failed, or Running) along with the [summary of the sync log](#sync-log-summary).  
+    Sync History displays the sync status or [reset](https://docs.airbyte.com/operator-guides/reset/) status (Succeeded, Partial Success, Failed, Cancelled, or Running) and the [sync summary](#sync-summary).
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud.md:119]: @Amruta-Ranade Do you think I should link to this Reset doc?

File: docs/cloud/managing-airbyte-cloud.md [hunk 2]
```diff
@@ -97,76 +106,45 @@ Understanding the following limitations will help you better manage Airbyte Clou
...
 
-3. Under Sync History, a list of syncs for that connection is displayed.
-
-    The sync status is displayed (Succeeded, Cancelled, Failed, or Running) along with the [summary of the sync log](#sync-log-summary).  
+    Sync History displays the sync status or [reset](https://docs.airbyte.com/operator-guides/reset/) status (Succeeded, Partial Success, Failed, Cancelled, or Running) and the [sync summary](#sync-summary).  
 
-4. For more information about a sync, click the sync log dropdown.  
+    :::note 
+    
+    Airbyte will try to sync your data three times. After a third failure, it will stop attempting to sync.
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud.md:123]: @Amruta-Ranade Do you think I should keep or delete this note about Airbyte syncing data three times before stopping?

**Comment by sophia-wiley** [comment]
@Amruta-Ranade this doc is ready for review :)

File: docs/cloud/managing-airbyte-cloud.md [hunk 3]
```diff
@@ -23,6 +23,15 @@ To add a user to your workspace:
     The user will have access to only the workspace you invited them to. Also note that they will be added as a workspace admin by default.
     :::
 
+### Remove users from your workspace​
+
+To remove a user from your workspace:
+
+1. On the Airbyte Cloud dashboard, click **Settings** in the navigation bar.
```
Comment (sophia-wiley) [inline docs/cloud/managing-airbyte-cloud.md:30]: @Amruta-Ranade should I add spaces between these list items for now until we restructure the doc?

File: docs/cloud/managing-airbyte-cloud.md (hunk 1 above)
Comment (Amruta-Ranade) [reply docs/cloud/managing-airbyte-cloud.md:125]: yep

File: docs/cloud/managing-airbyte-cloud.md (hunk 2 above)
Comment (Amruta-Ranade) [reply docs/cloud/managing-airbyte-cloud.md:129]: let's keep it for now

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**